wagtail/admin/tests/pages/test_revisions.py

```diff
@@ -432,6 +432,7 @@ def setUp(self):
             "<p>This year, to save me from tears, "
             "I'll give it to someone special</p>"
         )
+        self.christmas_event.go_live_at = local_datetime(2014, 12, 25)
```

Comment:
Setting this is more realistic, because that is usually set by the user in the admin and this triggers the scheduling of the revision.
Without that, the log entry would be something like `"Revision {self.this_christmas_revision.id} from {expected_created_at} unscheduled from publishing at None."`.

Actually, I've not any way to not set a go_live_at timestamp by using the admin interface. So maybe the tests are the only reason, why go_live_at may be None at the following check:
https://github.com/wagtail/wagtail/blob/0ddc9ff6c2189a2823d2ad91298d8132faf66843/wagtail/models/__init__.py#L2738-L2740

Reply:
@laymonage Thanks for the extensive review!
I hope I've understood your comments correctly: I would revert my above change, and also my test-additions below, and instead add/adapt the tests in those 2 mentioned modules (`wagtail.tests.test_audit_log` for the log data and `wagtail.admin.tests.test_audit_log` for the admin history views) to check the expected output (to contain the right timestamps).

Reply:
Yep, that's it! 👍